cmd/cloud-controller-manager/app/options/constant.go

```diff
@@ -0,0 +1,59 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package options
+
+// CloudControllerManagerUserAgent is the userAgent name when starting cloud-controller managers.
+const CloudControllerManagerUserAgent = "cloud-controller-manager"
+
+// DefaultInsecureCloudControllerManagerPort is the default insecure cloud-controller manager port.
+const DefaultInsecureCloudControllerManagerPort = 0
+
+// CloudControllerManagerGeneric flag sets generic cloud controller manager option.
+const CloudControllerManagerGeneric = "generic"
+
+// CloudControllerManagerHiddenMark sets hidden flag.
+const CloudControllerManagerHiddenMark = "controllers"
+
+// CloudControllerManagerServiceController cloud controller manager service controller name.
+const CloudControllerManagerServiceController = "service controller"
+
+// CloudControllerManagerSecureServing cloud controller manager secure serving option value.
+const CloudControllerManagerSecureServing = "secure serving"
+
+// CloudControllerManagerInsecureServing cloud controller manager insecure serving option value.
+const CloudControllerManagerInsecureServing = "insecure serving"
+
+// CloudControllerManagerAuthentication flag sets authentication headers.
+const CloudControllerManagerAuthentication = "authentication"
+
+// CloudControllerManagerAuthorization  flag sets authentication headers.
+const CloudControllerManagerAuthorization = "authorization"
+
+// CloudControllerManagerMisc initiate misc/additional flags.
+const CloudControllerManagerMisc = "misc"
+
+// CloudControllerManagerMaster flag sets the address of the Kubernetes API server.
+const CloudControllerManagerMaster = "master"
+
+// CloudControllerManagerKubeConfig flag sets Path to kubeconfig file with authorization and master location information.
+const CloudControllerManagerKubeConfig = "kubeconfig"
+
+// CloudControllerManagerNodeStatusUpdateFrequency flag sets specifies how often the controller updates nodes' status.
+const CloudControllerManagerNodeStatusUpdateFrequency = "node-status-update-frequency"
+
+// CloudControllerManagerInsecureServingNetworkProtocol defines the default insecure serving network protocol
+const CloudControllerManagerInsecureServingNetworkProtocol = "tcp"
```

Comment:
BTW, why not using 
```
const (
...
...
)
```
If work in this PR need, i will feedback the correct  description